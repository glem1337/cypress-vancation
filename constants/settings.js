import ROUTES from './routes';

// eslint-disable-next-line import/prefer-default-export
export const SETTING_LAYOUT_ITEMS = [
  {
    icon: 'profile',
    contentId: 'settingLayout.itemProfile',
    key: ROUTES.SETTINGS.PROFILE.KEY,
    path: ROUTES.SETTINGS.PROFILE.PATH,
  },
  {
    icon: 'account',
    contentId: 'settingLayout.itemAccount',
    key: ROUTES.SETTINGS.ACCOUNT.KEY,
    path: ROUTES.SETTINGS.ACCOUNT.PATH,
  },
  //* TODO - (For now, it must be hidden) :start
  // {
  //   icon: 'notifications-settings',
  //   contentId: 'settingLayout.itemNotifications',
  //   key: ROUTES.SETTINGS.NOTIFICATIONS.KEY,
  //   path: ROUTES.SETTINGS.NOTIFICATIONS.PATH,
  // },
  // {
  //   icon: 'wallet',
  //   contentId: 'settingLayout.itemPayment',
  //   key: ROUTES.SETTINGS.PAYMENT.KEY,
  //   path: ROUTES.SETTINGS.PAYMENT.PATH,
  // },
  //* TODO - (For now, it must be hidden) :end
];
