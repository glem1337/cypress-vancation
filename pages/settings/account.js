export { default } from 'views/settings/Account';
