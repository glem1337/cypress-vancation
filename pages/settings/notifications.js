export { default } from 'views/settings/Notifications';
