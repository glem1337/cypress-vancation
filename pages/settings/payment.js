export { default } from 'views/settings/Payment';
