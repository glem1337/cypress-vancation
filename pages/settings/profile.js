export { default } from 'views/settings/Profile';
