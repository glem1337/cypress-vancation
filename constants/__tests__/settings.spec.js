import * as settings from '../settings';

it('users constants matches snapshot', () => {
  expect(settings).toMatchSnapshot();
});
