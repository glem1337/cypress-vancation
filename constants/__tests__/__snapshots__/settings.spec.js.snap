// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`users constants matches snapshot 1`] = `
Object {
  "SETTING_LAYOUT_ITEMS": Array [
    Object {
      "contentId": "settingLayout.itemProfile",
      "icon": "profile",
      "key": "PROFILE",
      "path": "/settings/profile",
    },
    Object {
      "contentId": "settingLayout.itemAccount",
      "icon": "account",
      "key": "ACCOUNT",
      "path": "/settings/account",
    },
  ],
}
`;
